sqlglot/expressions.py

```diff
@@ -1185,7 +1185,7 @@ def to_dot(self) -> Dot:
                 parts.append(parent.expression)
             parent = parent.parent
 
-        return Dot.build(deepcopy(parts))
+        return Dot.build(deepcopy(parts)) if len(parts) > 1 else parts[0]
```

Comment:
This would previously break in `Dot.build` in cases where `parts` only had one item.